app/enterprise/0.32-x/oidc-google.md

```diff
@@ -58,9 +58,9 @@ standardized][oidc-standard-claims], however--if a provider returns an `email` c
 This also requires that clients login using an account mapped to some consumer, which may not be desirable (e.g. you apply OpenID Connect to a service, but only use plugins requiring a consumer on some routes). To deal with this, you can set the `anonymous` parameter in your OIDC plugin configuration to the ID of a generic consumer, which will then be used for all authenticated users that cannot be mapped to some other consumer.
 
 
-[add-certificate]: /latest/admin-api/#add-certificate
+[add-certificate]: /0.13.x/admin-api/#add-certificate
 [google-oidc]: https://developers.google.com/identity/protocols/OpenIDConnect
 [google-create-credentials]: https://console.developers.google.com/apis/credentials
-[add-api]: /enterprise/latest/getting-started/adding-your-api/
+[add-api]: /enterprise/{{page.kong_version}}/getting-started/adding-your-api/
 [oidc-id-token]: http://openid.net/specs/openid-connect-core-1_0.html#IDToken
```

Comment:
This section doesn't appear on the preview page.

Reply:
https://docs.konghq.com/enterprise/0.32-x/getting-started/adding-your-api/